src/imaginghelper.js

```diff
@@ -689,8 +689,17 @@
      **/
     function onOpen() {
         this._haveImage = true;
-        this.imgWidth = this._viewer.viewport.contentSize.x;
-        this.imgHeight = this._viewer.viewport.contentSize.y;
+        var contentSizeViewport = {},
+            contentSize;
+        // check if world is defined
```

Comment:
This comment isn't true anymore. I guess what it should say is a note that contentSize was deprecated in OSD 2.2.0, and that's why this check is being made.

As for actually checking for the existence of `world`, it's probably reasonable to assume that it exists if contentSize does not.